integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
I imagined `userData` table would look like this:
```js
{
   id: 'df23cc4a-1b66-11e5-b60b-1697f925ec7b",
   lastLogin: "Thursday, 25-Jun-15 18:21:03 UT",
   otherField: "..."
}
```
where `id` is a user `id`. I thought we would put miscellaneous user data in `userData` table and we can access them given a user `id` (we can do joins with `users` table because primary key of `userData` is a user `id`). 

To answer your question, we would use the `userData` table when we have access to a user's `id`.

For activation, we dont' have access to the user's `id`. We need to lookup a user `id` given `activationCode`. So If I were to put `activationCode` field in `userData` table, we need to have a secondary index on `userData` table for `activationCode` field. 

What do other people think? I'm open to adding `activationCode` into `userData` table.